libs/sfm/ba_sparse_matrix.h

```diff
@@ -256,6 +259,17 @@ template <typename T>
 SparseMatrix<T>
 SparseMatrix<T>::multiply (SparseMatrix const& rhs) const
 {
+#ifdef _OPENMP
+    return this->parallel_multiply(rhs);
```

Comment:
Parallelization only makes sense for reasonable large matrices, we could introduce a switch here.